src/Navbar.jsx

```diff
@@ -1,36 +1,44 @@
 import { Link } from 'react-router-dom';
 import PropTypes from 'prop-types';
+import { IconButton, Button } from '@chakra-ui/react';
+import {ChevronLeftIcon} from '@chakra-ui/icons';
+import { useNavigate } from 'react-router-dom';
 
-function Navbar({ isAdmin }) {
+// Navbar is a container with a single button for Contact Us
+
+function Navbar({ showContactUs, title, showSettings, excelDownload, backButton}) { // what is title
+  const navigate = useNavigate();
   const businessList = [
-    { name: 'Dashboard', path: '/BusinessDashboard' },
-    { name: 'Donation Tracking Form', path: '/BusinessDonationTrackingForm' },
-    { name: 'Business Onboarding', path: '/BusinessOnboardingForm' },
-    { name: 'Business Notification Center', path: '/BusinessNotificationCenter' },
+    { name: 'Contact Us', path: '/ContactUs' },
+    { name: 'Settings', path: '/EditContactInformation'},
+    { name: 'Excel Download', path: '/Download'}, // Download not made yet
+    { name: 'Back Button', path: '/'},
   ];
 
-  const adminList = [
-    { name: 'Dashboard', path: '/AdminDashboard' },
-    { name: 'All Businesses', path: '/AdminAllBusinesses' },
-    { name: 'Manage Forms', path: '/AdminManageForms' },
-    { name: 'Team Management', path: '/AdminTeamManagement' },
-  ];
+  //const navList = isAdmin ? adminList : businessList;
+  //console.log(navList);
+
+  const navList = businessList.filter(item => {
```

Comment:
🚫 **[eslint]** <[no-unused-vars](https://eslint.org/docs/latest/rules/no-unused-vars)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'navList' is assigned a value but never used.